modules/cardinal_service_blocks/cardinal_service_blocks.module

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * cardinal_service_blocks.module
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function cardinal_service_blocks_form_layout_builder_configure_block_alter(&$form, FormStateInterface $form_state, $form_id) {
+  /** @var \Drupal\layout_builder\SectionComponent $component */
+  $component = $form_state->getFormObject()->getCurrentComponent();
```

Comment:
Will getCurrentComponent() always return a valid object for $component?

Reply:
yeah. because of the nature of this hook it should always be that object. its altering the form `layout_builder_configure_block` which has that component